usmqe_tests/web/skyring/test_log.py

```diff
@@ -0,0 +1,92 @@
+"""
+Description: Simple log in test
+
+Author: ltrilety
+"""
+
+import pytest
+
+from selenium.webdriver.common.keys import Keys
+
+from webstr.selenium.ui.support import WebDriverWait, WaitForWebstrPage
+
+from usmqe.web.skyring.loginpage import pages as loginpage
+from usmqe.web.skyring.mainpage.navpage import pages as navpage
+
+
+# NOTE: example of test with log_in fixture used
+#       it should not be run hence not in the name
+def not_test_it(log_in, testcase_end):
+    """ example of test with log_in fixture,
+    with prepared log in outside of test
+    """
+    pytest.check(log_in.loginpage)
+    pytest.check(log_in.navbar)
+
+
+def test_positive_login(testcase_set, testcase_end):
+    """@usmid web/login_positive
+    Login as valid user.
+    """
+    loginpage_inst = loginpage.LoginPage(testcase_set.driver)
+    pytest.check(
+        loginpage_inst.login_user(
+            pytest.config.getini("usm_username"),
+            pytest.config.getini("usm_password")
+        ).__class__.__name__ == 'NavMenuBars',
+        "Login and wait for main page.")
+
+
+@pytest.mark.parametrize(("username", "password", "error_message"), [
+    ("none", "none", "Authentication Error!"),
+    ("", "", "The username and password cannot be blank."),
+    (True, "", "The username and password cannot be blank."),
+])
+def test_negative_login(testcase_set, testcase_end,
+                        username, password, error_message):
+    """
+    test negative login cases
+    """
+    if username is True:
+        username = pytest.config.getini("usm_username")
+    if password is None:
```

Comment:
I see that you don't have `password` set to `None` in the pytest.mark.parametrize list, is that intentional?